GDJS/package.json

```diff
@@ -16,7 +16,7 @@
     "minimist": "^1.2.5",
     "patch-package": "^6.4.7",
     "pixi.js": "7.3.0",
-    "prettier": "2.1.2",
+    "prettier": "^2.1.2",
```

Comment:
Did you really need this? We prefer having fixed dependencies to avoid using uncontrolled versions

Reply:
Probably not. I thought this change was a part of npm run format but I guess not. 
My git extension on vscode  is having issues and messing up changes/updates